assets/scss/partials/components/_notice.scss

```diff
@@ -0,0 +1,51 @@
+.notice {
+  padding-left: 0;
+  padding-right: 0;
+  width: 100%;
+  @include themed() {
+    background-color: t('notice-info');
+  }
+
+  &--update {
+    @include themed() {
+      background-color: t('notice-update');
+    }
+  }
+  &--warning {
+    @include themed() {
+      background-color: t('notice-warning');
+    }
+  }
+
+  &__title {
+    background-color: rgb(96, 134, 180);
+    float: right;
```

Comment:
Please avoid `float` to prevent us headaches. If you don't see why, here is an [article](https://css-tricks.com/is-css-float-deprecated/). You can realize this easily with flexbox.

```SCSS
.notice {
  display: flex;
  flex-direction: column; 

  &__title {
    align-self: flex-end;
  }
```